rllib/tuned_examples/dt/pendulum-v1-medium-expert-dt.yaml

```diff
@@ -0,0 +1,48 @@
+pendulum_medium_expert_dt:
+    env: 'Pendulum-v1'
+    run: DT
+    stop:
+        # We could make this -200, but given that we have 4 cpus for our tests, we will have to settle for -300.
+        evaluation/episode_reward_mean: -350
```

Comment:
why -350 then?

Reply:
oh the comment is a typo, should be -350. -300 also works but it'll take longer to train because this is medium-expert

Reply:
fixed